jenkins/internal-ci/generic/pr/cortx_deployment/pyutils.groovy

```diff
@@ -237,11 +239,50 @@ pipeline {
                 }
             }
         }
+
+        stage ("Test Utils") {
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                script {
+                        NODE_HOST = sh( script: '''
+                            echo $host | tr ' ' '\n' | head -1 | awk -F["="] '{print $2}' | cut -d',' -f1    
+                        ''', returnStdout: true).trim()
+                        NODE_USER = sh( script: '''
+                            echo $host | tr ' ' '\n' | head -1 | awk -F["="] '{print $3}' | cut -d',' -f1    
+                        ''', returnStdout: true).trim()
+                        NODE_PASS = sh( script: '''
+                            echo $host | tr ' ' '\n' | head -1 | awk -F["="] '{print $4}' | cut -d',' -f1    
+                        ''', returnStdout: true).trim()
+                        def remote = getRemoteMachine(NODE_HOST, NODE_USER, NODE_PASS)
+                        def commandResult = sshCommand remote: remote, command: '''
+                            DATA_POD="$(( kubectl get pods | grep "cortx-data" | awk '{print $1}' ) 2>&1)"
+                            echo $DATA_POD
+                            kubectl exec $DATA_POD --container cortx-hax -- bash -c 'yum install -y wget yum-utils \
+                                && BUILD=$(grep BUILD < RELEASE.INFO | awk -F["\\""] "{ print \\$2 }" | tr -d "\\"") \
```

Comment:
Can we use `BUILD=$(grep BUILD < RELEASE.INFO | awk '{print $2}' | sed 's|"||g')`

Reply:
Done